src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -4770,11 +4770,19 @@ private static bool CanReuseVariableDeclarator(CSharp.Syntax.VariableDeclaratorS
                                 currentTokenKind == SyntaxKind.MinusGreaterThanToken ||
                                 isNonEqualsBinaryToken)
                             {
-                                var missingIdentifier = CreateMissingIdentifierToken();
-                                missingIdentifier = this.AddError(missingIdentifier, offset, width, ErrorCode.ERR_IdentifierExpected);
+                                var isPossibleLocalFunctionToken =
+                                    currentTokenKind == SyntaxKind.OpenParenToken |
```

Comment:
Oh, I get it.

---
In reply to: [76164551](https://github.com/dotnet/roslyn/pull/13263#discussion_r76164551) [](ancestors = 76164551,76164331)